renter/proto/session.go

```diff
@@ -727,10 +739,15 @@ func newUnlockedSession(hostIP modules.NetAddress, hostKey hostdb.HostPublicKey,
 	}
 	latency := time.Since(start)
 	conn := &statsConn{Conn: tcpConn}
-	conn.SetDeadline(time.Now().Add(60 * time.Second))
+	err = conn.SetDeadline(time.Now().Add(60 * time.Second))
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Practically speaking, I don't think it's possible for an error to be returned here. Looking at the source, `(TCPConn).SetDeadline` will only return an error if the underlying file descriptor is closed, and we *know* that it's open here. And since this is a `TCPConn`, we don't need to worry about other types that implement `SetDeadline` with weird behavior.

However, it's possible that the implementation of `TCPConn` could change in the future. Furthermore, I will probably allow arbitrary `net.Conn`s in `Session` soon (as per #92). And in the worst case, failing to set a deadline could cause a `Read` or `Write` to block indefinitely. So I suppose this error should be handled properly.